docs/reference/how-to/size-your-shards.asciidoc

```diff
@@ -221,25 +221,40 @@ GET _cat/shards?v=true
 // TEST[setup:my_index]
 
 [discrete]
-[[field-count-recommendation]]
-==== Data nodes should have at least 1kB of heap per field per index, plus overheads
+[[fields-heap-recommendation]]
```

Comment:
These IDs form part of URLs so it's best not to change them - for instance renaming this ID will break any links to https://www.elastic.co/guide/en/elasticsearch/reference/current/size-your-shards.html#field-count-recommendation